Dockerfiles/midstream/unit_tests.py

```diff
@@ -31,7 +31,7 @@ def get_error_message_content(output_file=".errormessage"):
 
     # Running the container with correctly set environment variables, default channel missing, all tasks should pass
     def test_positive_missing_default_channel(self):
-        self.env["IMAGE_TO_TEST"] = "quay.io/cvpops/test-operator:missing-default-channel-v1"
+        self.env["IMAGE_TO_TEST"] = "quay.io/cvpops/test-operator:test-default-positive-v1"
```

Comment:
@dirgim is the image you gave me missing the default channel, or will I need to build a different image for this test

Reply:
It's not missing a default channel, but I built one that does (while also passing the deprecated API check) - quay.io/cvpops/test-operator:missing-default-channel-v2